src/test/java/orangetest/pages/VacanciesPage.java

```diff
@@ -0,0 +1,84 @@
+package orangetest.pages;
+
+import orangetest.data.Vacancy;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class VacanciesPage extends BasePage {
+
+    @FindBy(xpath = "//input[@id='btnAdd']")
+    WebElement btnAdd;
+
+    @FindBy(xpath = "//input[@id='btnSave']")
+    WebElement btnSave;
+
+    @FindBy(xpath = "//select[@id='addJobVacancy_jobTitle']")
+    WebElement jobTitle;
+
+    @FindBy(xpath = "//input[@id='addJobVacancy_name']")
+    WebElement vacancyName;
+
+    @FindBy(xpath = "//input[@id='addJobVacancy_hiringManager']")
+    WebElement hiringManager;
+
+    @FindBy(xpath = "//input[@id='addJobVacancy_noOfPositions']")
+    WebElement numberOfPositions;
+
+    @FindBy(xpath = "//table[@id='resultTable']//tbody")
+    WebElement resultTable;
+
+    @FindBy(css = "div.box:nth-child(1) div.inner form:nth-child(1) fieldset:nth-child(3) ol:nth-child(1) li:nth-child(1) > span.validation-error:nth-child(3)")
+    WebElement errorJobTitle;
+
+    @FindBy(css = "div.box:nth-child(1) div.inner form:nth-child(1) fieldset:nth-child(3) ol:nth-child(1) li:nth-child(2) > span.validation-error:nth-child(3)")
+    WebElement errorVacancyName;
+
+    @FindBy(css = "div.box:nth-child(1) div.inner form:nth-child(1) fieldset:nth-child(3) ol:nth-child(1) li:nth-child(3) > span.validation-error:nth-child(3)")
+    WebElement errorHiringManager;
+
+    private Vacancy vacancy = new Vacancy();
+
+    public VacanciesPage() {
+        super();
+    }
+
+    public void addVacancy() {
+        menuRecruitment.click();
+        viewJobVacancy.click();
+        btnAdd.click();
+        jobTitle.click();
+        jobTitle.findElement(By.xpath("//option[contains(text(),'AQA')]")).click();
+        vacancyName.sendKeys(vacancy.getVacancyName());
+        hiringManager.sendKeys(vacancy.getHiringManager());
+        numberOfPositions.sendKeys(vacancy.getNumberOfPositions());
+        btnSave.click();
+    }
+
+    public boolean addEmptyVacancy() {
+        menuRecruitment.click();
```

Comment:
Method name that returns boolean shoud start with `is` or `has`. Please rename all methods where it is.

Reply:
fixed